codebase-stewardship/index.html

```diff
@@ -27,13 +28,21 @@ <h1>Everything your project needs to be sustainable, collaborative and open.</h1
 <img class="hero" alt="illustration, codebase stewardship" src="https://publiccodenet.github.io/illustrations/illustrations/codebase-2.svg">
 
 <div class="mp-section">
+
   <div><h2>Making sure the code, tests and documentation are always up to the standards</h2>
 
   <p>
     We check every merge request on code, policy and documentation quality in our <strong>codebase auditing</strong> process. This enables agile development for trustworthy codebases for public organizations and their collaborators.
   </p>
 
   </div>
+
+  <blockquote>
+    <img src="../photos/mirjam-hierarchy.jpg" alt="Martin Dias d'Ullois" class="testimonial-photo">
```

Comment:
Can we do this with a generic illustration or a grey circle? Maybe a photo of the Logius building or something?

Reply:
[Logius uses the overheid crest](https://twitter.com/logius_minbzk) as their logo - do you think we can just borrow that?